WinFormsUI/DisplayManager.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Core;
+
+namespace WinFormsUI
+{
+	public class DisplayManager
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5924375)